Client/multiplayer_sa/multiplayer_shotsync.cpp

```diff
@@ -168,6 +174,16 @@ VOID WriteGunDirectionDataForPed(CPedSAInterface* pPedInterface, float* fGunDire
             // Make sure our pitch is updated (fixes first-person weapons not moving)
             *fGunDirectionY = pGameInterface->GetCamera()->Find3rdPersonQuickAimPitch();
 
+            // Are we doing driveby?
+            if (pAimingPed->IsDoingGangDriveby())
+            {
+                // Fix pitch in driveby when facing left or backwards
+                if (LocalShotSyncData.m_cInVehicleAimDirection == 1 || LocalShotSyncData.m_cInVehicleAimDirection == 2)
+                {
+                    *fGunDirectionY = -*fGunDirectionY;
+                }
```

Comment:
I can see magic numbers here. It would be nice if you can define an enum in CMultiplayer.h. 

```
enum class eVehicleAimDirection : char
{
    FORWARDS = 0,
    LEFT, 
    BACKWARDS,
    RIGHT
};
```
CShotSyncData:
```
 eVehicleAimDirection m_cInVehicleAimDirection;
```

The fix:
```
switch (LocalShotSyncData.m_cInVehicleAimDirection)
{
    case eVehicleAimDirection::LEFT:
    case eVehicleAimDirection::BACKWARDS:
    *fGunDirectionY = -*fGunDirectionY;
    break;
}
```
You would also need to change a few references in multiplayer_shotsync.cpp from char to eVehicleAimDirection in multiplayer_shotsyn.cpp, CClientGame, and CClientPed. The compiler will throw errors and let you know where to change them.